### PR TITLE
Avira-VPN: Added new brew cask for Avira VPN.

### PR DESCRIPTION
**Pull request by SomethingElese18878**
#### Adding a new cask
- [ ] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [ ] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [ ] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] Commit message includes cask’s name.
- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` left no offenses.
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by dasSOZO** [comment]
I am confused how I should fix the url in the comment to pass the automated test. Can anyone help?

**Comment by vitorgalvao** [comment]
You need to actually fill the template.

File: Casks/avira-vpn.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+  # avira-update.com was verified as official when first introduced to the cask
+  url 'https://install.avira-update.com/package/vpn/osx/int/vpn.pkg'
+  name 'Avira VPN'
+  homepage 'http://www.avira.com/en/avira-phantom-vpn'
+  license :commercial
+
+  pkg 'vpn.pkg'
+
+  uninstall delete:  '/Applications/Avira Phantom VPN.app'
```
Comment (vitorgalvao) [inline Casks/avira-vpn.rb]: It’s doubtful this is the only thing the `pkg` does. There are [scripts](https://github.com/caskroom/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/uninstall.md) geared for finding all it needs.

**Comment by jawshooah** [comment]
Thank you for the contribution. It needed some fixes, so they were made in e6abfbdd3ade3314ecd6535ae4ed6ca775e93324. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**